save_command_strategies/cmdline.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+PANE_PID="$1"
+CPID=$(pgrep -P $PANE_PID)
+
+exit_safely_if_empty_ppid() {
+	if [ -z "$PANE_PID" ]; then
+		exit 0
+	fi
+}
+
+full_command() {
+	[[ -z "$CPID" ]] && exit 0
+	cat /proc/${CPID}/cmdline | xargs -0 printf "%q "
```

Comment:
I would expect `/proc/${CPID}/cmdline` to output a single line.. so can you please clarify why we need `| xargs -0 printf "%q "`?

Reply:
It contains a null-terminated set of strings representing each argument, which are then separated into individual arguments by `xargs` and passed to `printf` to be quoted.  This makes it so arguments with spaces are preserved